frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.tsx

```diff
@@ -78,6 +78,17 @@ export const CapacityCard: React.FC<DashboardItemProps> = ({
   const storageTotal = prometheusResults.getIn([matchingQueries[0], 'result']);
   const storageUsed = prometheusResults.getIn([matchingQueries[1], 'result']);
 
+  let statUsed: React.ReactText = getLastStats(storageUsed, getInstantVectorStats);
+  let statTotal: React.ReactText = getLastStats(storageTotal, getInstantVectorStats);
+  // for 'CAPACITY_TOTAL_USAGE', we get 'data.result[0].values' *NOT* value property
+  // which has array/vector of values and will return 'NaN', will use 'getRangeVectorStats'
+  if (Number.isNaN(Number(statUsed))) {
```

Comment:
I think we will not need this extra logic if te timer is removed.

Reply:
Acknowledged...